e2etests/cloud/run_cloud_drift_test.go

```diff
@@ -65,6 +65,10 @@ func TestCLIRunWithCloudSyncDriftStatus(t *testing.T) {
 		tofu          bool
 	}
 
+	makeSerial := func(serial int64) *int64 {
```

Comment:
Isn't it possible to have a test with a serial different from 0?

Reply:
I also think this would very good but it will require calling apply in the test and last time I did this I regretted due to the slowness of it.
Maybe if we commit an initial statefile but this also has some problems because the state depends on the TF version...

Reply:
Done in [191a989](https://github.com/terramate-io/terramate/pull/1783/commits/191a989e8e1d726ea5c0c3b5d46b47f3b06d6b17) Awesome!